app/uk/gov/hmrc/eusubsidycompliancefrontend/views/UndertakingAddBusinessPage.scala.html

```diff
@@ -70,12 +70,12 @@
         RadioItem(
           content = Text(messages("common.yes")),
           value = Some("true"),
-          checked = form.data.get(fieldName).fold(false)(e => e)
+          checked = form.data.get(fieldName).fold(false){e => e == "true"}
```

Comment:
Probably nice if there was a test for this logic but currently this logic is not really applied  as it never can invalidate with a value